packages/daemon/src/veramo/__tests__/vc-management.ts

```diff
@@ -0,0 +1,20 @@
+// import jest from 'jest';
+import { initVeramo, _clearVeramo, findDaemonDid } from '../did-manager';
+
+describe('did-manager', () => {
+  let agent = null;
+
+  beforeAll(async () => {
+    ({ agent } = await initVeramo({
+      dbFile: 'database.test.sqlite',
+      veramoSecret: '29739248cad1bd1a0fc4d9b75cd4d2990de535baf5caadfdf8d8f86664aa830c',
+      infuraProjectId: '9386bd3e92034b6784bc17ed1d7253f4',
+    }));
+  })
+  afterAll(_clearVeramo);
+
+  it('should create Daemon service did on start', async () => {
+    const daemonDid = await findDaemonDid();
+    expect(daemonDid).not.toBeNull();
+  })
```

Comment:
Now able to write more of these kinds of tests exercising the backend lib and verifying that it does the right thing